lib/src/sql/statement.rs

```diff
@@ -208,9 +213,9 @@ pub fn statement(i: &str) -> IResult<&str, Statement> {
 			map(remove, Statement::Remove),
 			map(select, Statement::Select),
 			map(set, Statement::Set),
+			map(show, Statement::Show),
 			map(sleep, Statement::Sleep),
-			map(update, Statement::Update),
-			map(yuse, Statement::Use),
+			alt((map(update, Statement::Update), map(yuse, Statement::Use))),
```

Comment:
Why do we need this change?

Reply:
@emmanuel-keller Hey! That's because `alt` supports only up to 21 alternatives. This workaround of nesting `alt`s for more is documented in the nom doc! here's the quick reference to the doc addition PR https://github.com/rust-bakery/nom/commit/6c93eeaca17c227b7ad33822b3c53e5282bf900e